audit/audit-template.yml

```diff
@@ -82,6 +82,46 @@ Resources:
               ArnLike:
                 "kms:EncryptionContext:aws:logs:arn": !Sub "arn:aws:logs:${AWS::Region}:${AWS::AccountId}:*"
 
+  FireHoseKMSKey:
+    Type: AWS::KMS::Key
+    Properties:
+      EnableKeyRotation: true
+      KeyPolicy:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: "Allow"
+            Principal:
+              AWS: !Sub "arn:aws:iam::${AWS::AccountId}:root"
```

Comment:
Why do we state "aws" here and not reference ${AWS::Partition}?
I'm pretty sure it doesn't matter, but was just curious

Reply:
We could use it. It would end up being the same. It's more useful I imagine when deploying to regions that make use of it. Such as Government partitions in America.